common-lib/src/main/scala/com/gu/mediaservice/model/UsageRights.scala

```diff
@@ -396,5 +397,12 @@ case class CreativeCommons(restrictions: Option[String] = None)
   }
 object CreativeCommons {
  implicit val jsonReads: Reads[CreativeCommons] = Json.reads[CreativeCommons]
- implicit val jsonWrites: Writes[CreativeCommons] = UsageRights.defaultWrites
+ implicit val jsonWrites: Writes[CreativeCommons] = (
```

Comment:
As the `UsageRights.defaultRights` can only deals with category and restrictions.
I have a branch looking at composing said function (or using partials) which cleans up a lot of this manual labour - I just haven't had time to completely finish / figure it out.